src/module/index.ts

```diff
@@ -5,6 +5,7 @@ import defaults from "./defaults";
 import merge from "lodash.merge";
 import createMiddleware, { CustomRoute } from "../api";
 import { FSXAApi, FSXAContentMode, LogLevel } from "fsxa-api";
+import { json } from "express";
```

Comment:
Is there a reason why you added this import?

Reply:
ahh just an auto import i didnt see, this can be removed